lib/svg4everybody.js

```diff
@@ -94,13 +94,16 @@ function svg4everybody(rawopts) {
 	var newerIEUA = /\bTrident\/[567]\b|\bMSIE (?:9|10)\.0\b/;
 	var webkitUA = /\bAppleWebKit\/(\d+)\b/;
 	var olderEdgeUA = /\bEdge\/12\.(\d+)\b/;
+	var edgeUA = /\bEdge\/.(\d+)\b/;
+	//Source: http://stackoverflow.com/questions/21751377/foolproof-way-to-detect-if-this-page-is-inside-a-cross-domain-iframe
```

Comment:
minor: rather than a chunky source comment like this, can we just have something short like ... `// checks whether iframed `?